app/components/SphinxContainer.jsx

```diff
@@ -0,0 +1,63 @@
+
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Sphinx from './Sphinx';
+import Riddle from './Riddle';
+import {addImage,updateGuessed} from '../reducers/riddle';
+
+
+class SphinxContainer extends Component {
```

Comment:
This should be a presentational component as nothing is being stored on local state